build.sbt

```diff
@@ -5,6 +5,9 @@ scalaVersion := "2.12.4"
 val akkaHttpVersion = "10.1.0"
 val akkaVersion = "2.5.11"
 
+scapegoatVersion in ThisBuild := "1.3.8"
```

Comment:
Why are we putting the version for scapegoat in here, and not in libraryDependencies?

Reply:
it's a plugin for the build tool, not a dependency for code compilation

Reply:
sbt uses this version to know what code to use when you run the `scapegoat` task or any other added from the plugin itself